src/css/components/modal.css

```diff
@@ -0,0 +1,71 @@
+.display-none {
+  display: none;
+}
+.display-block {
+  display: block;
+}
+.modal {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  background-color: #00000059;
+  z-index: 99999;
+}
+
+.modal-container {
+  background: #fff;
+  max-width: 500px;
+  padding: 20px;
+  margin: auto;
+  border-radius: 5px;
+  position: relative;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  animation: animatetop 0.4s ease-in-out;
+  -webkit-animation-name: animatetop;
+  -webkit-animation-duration: 0.5s;
+  animation-name: animatetop;
+  animation-duration: 0.5s;
+  position: absolute;
+  left: 50%;
+  top: 40%;
+  transform: translate(-50%, -50%);
+  width: 98%;
+}
+.close-btn {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  font-size: 1.3rem;
+  margin: auto;
+  box-shadow: 0px 0px 6px 0 rgba(0, 0, 0, 0.2),
+    0px 2px 10px 0 rgba(0, 0, 0, 0.19);
+  border: none;
+  background: white;
+  color: #c04239;
+}
+/* Add Animation */
+@-webkit-keyframes animatetop {
+  from {
+    top: -300px;
+    opacity: 0;
+  }
+  to {
+    top: 40%;
+    opacity: 1;
+  }
+}
+
+@keyframes animatetop {
```

Comment:
Love that you took the care to animate in your modal!!!

Not blocking, but something to consider is that animating a `translate3d` can result in smoother tweens than animating properties such as `top` or `left`. It has to do with how the browser renders the elements. Any of the 3D functions (`translate3d`, `translateZ`, and strangely enough `opacity`) cause the animation to be offloaded to the computer’s GPU rather than relying on the CPU to do the necessary calculations. It’s usually _fine_ on modern browsers + capable devices, but if you have users on older hardware it could make a noticeable difference.

Reply:
yeah 3d will be smoother